flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -230,30 +230,34 @@ function start_local_zk {
     done < <(grep "^server\." "${FLINK_DIR}/conf/zoo.cfg")
 }
 
-function wait_dispatcher_running {
+function wait_rest_endpoint_up {
+  local port=$1
+  local endpoint=$2
```

Comment:
I think as a variable name `path` is better suited (instead of `endpoint`): https://en.wikipedia.org/wiki/URL